src/libs/ajax.js

```diff
@@ -95,16 +95,16 @@ export const DAC = {
     return res.json();
   },
 
-  create: async (name, description) => {
+  create: async (name, description, email) => {
     const url = `${await getApiUrl()}/api/dac`;
-    const dac = { 'name': name, 'description': description };
+    const dac = { name, description, email };
```

Comment:
The back end considers email optional - not sure if we've tested that case (I haven't). Might be worth some logic around the construction of the `dac` object.